app/src/main/kotlin/com/simplemobiletools/calculator/compose/extensions/BooleanPreviewParameterProvider.kt

```diff
@@ -0,0 +1,8 @@
+package com.simplemobiletools.calculator.compose.extensions
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+
+class BooleanPreviewParameterProvider : PreviewParameterProvider<Boolean> {
```

Comment:
what does this do?

Reply:
https://developer.android.com/jetpack/compose/tooling/previews#preview-data